pages/views/blogs/Contribution Needed - Docs.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
Note for reviewers: that one is not a stub. So feel free to make notes regarding content